.github/workflows/conda-cpp-tests.yaml

```diff
@@ -68,11 +68,10 @@ jobs:
           export MATRICES="
             pull-request:
               # amd64
-              - { ARCH: 'amd64', PY_VER: '3.9',  CUDA_VER: '11.4.3', LINUX_VER: 'centos7',      GPU: 'v100', DRIVER: 'latest'  }
-              - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu22.04',  GPU: 'v100', DRIVER: 'latest'  }
-              - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu22.04',  GPU: 'v100', DRIVER: 'latest'  }
+              - { ARCH: 'amd64', PY_VER: '3.9',  CUDA_VER: '11.8.0', LINUX_VER: 'centos7',     GPU: 'v100', DRIVER: 'earliest'   }
```

Comment:
Ooh, this rebalances one of the C++ jobs to use the "earliest" pool of runners. That's probably a net win.